templates/prometheus.yml.j2

```
@@ -97,3 +97,11 @@ scrape_configs:
       - target_label: __address__
         replacement: blackbox:{{ monitoring_blackbox_port }}
 {% endif %}
+
+{% if monitoring_postgresql_enabled | default(true) %}
+  - job_name: "postgresql"
+    static_configs:
+      - targets: ['{{monitoring_pgsql_db_host}}:{{monitoring_pgsql_exporter_port}}']
```

```suggestion
      - targets: ['{{ monitoring_pgsql_db_host }}:{{ monitoring_pgsql_exporter_port }}']
```
do not forget to run linter)